test/functional/integreatly_test.go

```diff
@@ -19,14 +19,15 @@ func TestIntegreatly(t *testing.T) {
 	t.Run("Integreatly Happy Path Tests", func(t *testing.T) {
 		for _, test := range common.ALL_TESTS {
 			t.Run(test.Description, func(t *testing.T) {
-				testingContext, err = common.NewTestingContext(config)
```

Comment:
Why are we deleting this?

Reply:
We're passing it from outside of the `for` loop from what I can see, we can probably do the same with the `functional loop` to avoid code repetition

Reply:
I think we should add this back in. Each test that we run should have its own testing context. This will ensure each test does not modify or change the context on another test. It will also ensure that every test run will have the latest/most up to date schemes/apis. 

So yeah, I think for `ALL_TEST` , `AFTER_INSTALL_TESTS` and `FUNCTIONAL_TESTS` we should add this back in.